app/views/staff/activity_forms/level.html.haml

```diff
@@ -0,0 +1,8 @@
+= render layout: "wrapper" do |f|
+  = f.hidden_field :level, value: nil
+  = f.govuk_collection_radio_buttons :level,
+    create_activity_level_options(user: current_user),
+    :level,
+    :description,
+    legend: { tag: 'h1', size: 'xl', text: I18n.t("form.legend.activity.level") },
+    hint_text: I18n.t("form.hint.activity.level").html_safe
```

Comment:
What is the concern that drives the use of `html_safe` here?

Reply:
Guilty of copy and pasting from another radio step! I presume that one has or had links to IATI in their hints. This can probably go 👍